resources/application-connector/charts/application-broker/templates/deployment.yaml

```diff
@@ -24,14 +26,16 @@ spec:
         sidecar.istio.io/rewriteAppHTTPProbers: "true"
       labels:
         app: {{ .Chart.Name }}
-        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-        release: "{{ .Release.Name }}"
-        heritage: "{{ .Release.Service }}"
+        release: {{ .Release.Name }}
+        helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+        app.kubernetes.io/name: {{ template "name" . }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ .Chart.Name }}
       containers:
       - name: ctrl
-        image: "{{ .Values.global.containerRegistry.path }}/{{ .Values.global.application_broker.dir }}application-broker:{{ .Values.global.application_broker.version }}"
+        image: "{{ .Values.global.containerRegistry.path }}/application-broker:{{ .Values.global.application_broker.version }}"
```

Comment:
Is it official we stop using `dir` like `develop` or `pr`?

Reply:
We do not use it for about a year. I asked about it on Slack and nobody minded. So I changed it because it seems to be redundant